native/http-rs-axum/Kraftfile

```diff
@@ -0,0 +1,26 @@
+spec: v0.6
```

Comment:
I guess it'd be good to change to `v0.7` as it is the latest spec. Doesn't change anything in the Kraftfile.

Reply:
`v0.6` is what is used in all the other examples in this repo, though. @razvand has also requested using `v0.6` instead of `v0.7` in the past (https://github.com/unikraft/catalog/pull/54#discussion_r1518967534). What should we do?

Reply:
Ah okay then, I just thought it was better to use the latest version. I didn't notice all the other entries were using 0.6. I suppose we should leave it like that then